gen/builders/tipsets.go

```diff
@@ -50,8 +54,8 @@ func (tss *TipsetSeq) All() []*Tipset {
 // Messages returns all ApplicableMessages that have been included in blocks,
 // in no particular order.
 func (tss *TipsetSeq) Messages() []*ApplicableMessage {
-	msgs := make([]*ApplicableMessage, 0, len(tss.msgIdx))
-	for _, msg := range tss.msgIdx {
+	msgs := make([]*ApplicableMessage, 0, len(tss.orderedMsgs))
+	for _, msg := range tss.orderedMsgs {
 		msgs = append(msgs, msg)
 	}
 	return msgs
```

Comment:
This was here to convert the map into a slice, but now we might as well return the `orderedMsgs` slice. Although if we want to keep it safe, we can simplify this using the builtin `copy()`.